std/process.d

```diff
@@ -1443,6 +1499,33 @@ int wait(Pid pid) @safe
     else version (Posix) assert(pid.osHandle < 0);
 }
 
+@system unittest // Pid.fromSystemPid()
+{
+    version (Windows)    TestScript prog = "do { timeout 1 } while (1)";
```

Comment:
In addition to fixing the script, you need to find why your test was broken (in that it allowed a broken script to work and pass the test).